src/Symfony/Bundle/FrameworkBundle/CHANGELOG.md

```diff
@@ -12,10 +12,11 @@ CHANGELOG
  * Add `DomCrawlerAssertionsTrait::assertAnySelectorTextNotContains(string $selector, string $text)`
  * Deprecate `EnableLoggerDebugModePass`, use argument `$debug` of HttpKernel's `Logger` instead
  * Deprecate `AddDebugLogProcessorPass::configureLogger()`, use HttpKernel's `DebugLoggerConfigurator` instead
- * Deprecate not setting the `handle_all_throwables` config option. The default value will change to `true` in 7.0
- * Deprecate not setting the `php_errors.log` config option. The default value will change to `true` in 7.0
- * Deprecate not setting the `session.cookie_secure` config option. The default value will change to `auto` in 7.0
- * Deprecate not setting the `session.cookie_samesite` config option. The default value will change to `lax` in 7.0
+ * Deprecate not setting the `framework.handle_all_throwables` config option. The default value will change to `true` in 7.0
+ * Deprecate not setting the `framework.php_errors.log` config option. The default value will change to `true` in 7.0
+ * Deprecate not setting the `framework.session.cookie_secure` config option. The default value will change to `auto` in 7.0
+ * Deprecate not setting the `framework.session.cookie_samesite` config option. The default value will change to `lax` in 7.0
+ * Deprecate not setting the `framework.session.handler_id` config option. It will default to `session.handler.native_file` if `framework.session.save_path` is set or `null` otherwise in 7.0
```

Comment:
```suggestion
 * Deprecate not setting the `framework.session.handler_id` config option; it will default to `session.handler.native_file` when `framework.session.save_path` is set or `null` otherwise in 7.0
```